microsoft-integration/admin-tool/src/main/resources/Messages_eu.properties

```diff
@@ -144,7 +147,8 @@ credentials.authority=Authority
 credentials.clientId=Client ID
 credentials.secret=Secret
 credentials.scope=Scope
-credentials.scope_alt=By default: https://graph.microsoft.com/.default
+credentials.delegated_scope = Delegated Scope
```

Comment:
@Pablosanroman would you mind translating these strings? Thanks

Reply:
credentials.clientId=Bezeroaren IDa
credentials.secret=Sekretua
credentials.scope=Irismena
credentials.scope_alt=Lehenetsia: https://graph.microsoft.com/.default
credentials.delegated_scope = Irismen delegatua